feature/runtime/testing/assert_test.lua

```diff
@@ -0,0 +1,25 @@
+local t = require("testing")
+
+t.run("actually true", function()
```

Comment:
I'm just using Lua closures here, but they can be established functions passed by name too.